rpm/nemo-qml-plugin-dbus-qt5.spec

```diff
@@ -20,25 +23,46 @@ Requires:   %{name} = %{version}-%{release}
 Requires:   qt5-qtdeclarative-import-qttest
 Requires:   qt5-qtdeclarative-devel-tools
 
+%package testd
```

Comment:
The existing tests package pulls in lots of things that are not needed when running only the manual test case. Also the daemon might be useful for something else too.

But all things considered having just one package is probably for the best.